kerl

```diff
@@ -2323,6 +2356,31 @@ case "$1" in
             fi
         fi
         ;;
+    build-install)
+        third="$3"
```

Comment:
can you please use more descriptive names here like `install_dir_or_x` or `build_name_or_y` where x and y are descriptive

Reply:
Sure thing... I'll add that to the list of actions in the description.

Reply:
Pushed.